ovs/lib/setup.py

```diff
@@ -1051,6 +1219,48 @@ def _setup_extra_node(cluster_ip, master_ip, cluster_name, unique_id, ip_client_
         logger.info('Extra node complete')
 
     @staticmethod
+    def _rollback_setup_extra_node(target_client):
```

Comment:
Can this method be merged (or at least partially) with rollback_first_node? I assume they will contain duplicate code.